js/stores/SectionStore.js

```diff
@@ -6,99 +6,192 @@ var assign = require('object-assign');
 var CHANGE_EVENT = 'change';
 
 var sections = {
-    "1a": {
-        "id": "1a",
-        "title": "LITURGIES",
-        "description": "\"This is My body which is given for you; My blood, which is shed for you.\" (Luke 22:19-20)",
-        "img": "../../images/placeholder.png",
+  "1a": {
+    "id": "1a",
+    "title": "LITURGIES",
+    "description": "\"This is My body which is given for you; My blood, which is shed for you.\" (Luke 22:19-20)",
+    "img": "../../images/placeholder.png",
+    "node": {
+      "1b": {
+        "id": "1b",
+        "title": "ST. BASIL",
         "node": {
-            "1b": {
-                "id": "1b",
-                "title": "ST. BASIL",
-                "node": {
-                    "1c": {
-                        "id": "1c",
-                        "title": "OFFERING OF THE LAMB",
-                        "load": "stbasil_offering"
-                    },
-                    "1d": {
-                      "id": "1d",
-                      "title": "LITURGY OF THE WORD",
-                      "load" : "stbasil_word"
-                    },
-                },
-
-            },
-            "2b": {
-                "id": "2b",
-                "title": "VESPERS",
-                "load": "vespers"
-            },
-            "3b": {
-                "id": "3b",
-                "title": "MATINS",
-                "load": "matins"
-            }
+          "1c": {
+            "id": "1c",
+            "title": "OFFERING OF THE LAMB",
+            "load": "stbasil_offering"
+          },
+          "1d": {
+            "id": "1d",
+            "title": "LITURGY OF THE WORD",
+            "load" : "stbasil_word"
+          },
+          "1e": {
+            "id" : "1e",
+            "title": "LITURGY OF THE FAITHFUL",
+            "load": "stbasil_faithful"
+          },
+
+        },
+
+      },
+      "2b": {
+        "id": "2b",
+        "title": "VESPERS",
+        "load": "vespers"
+      },
+      "3b": {
+        "id": "3b",
+        "title": "MATINS",
+        "load": "matins"
+      }
+    }
+  },
+  "2a": {
+    "id": "2a",
+    "title": "BOOK OF HOURS",
+    "description": "\"This is My body which is given for you; My blood, which is shed for you.\" (Luke 22:19-20)",
```

Comment:
Nice work! Go ahead and delete the description and img keys. Looks like your indentations are 2 spaces instead of 4. See if you can change that.

The 'ST' in "1ST" is capitalized while the titles have that part lower cased. Also "mIdnight" is misspelled.